functions/donation/thank-you.ts

```diff
@@ -32,27 +32,79 @@ const hasThankYouEnvVars = (env: unknown): env is ThankYouEnvVars => {
 }
 
 const constructMessage = (data: MessageData): string => {
+    // This is more readable and makes the plain text formatting easier than a
+    // template string.
     return [
-        `Dear ${data.name ? data.name : "Sir/Madam"},`,
-        "Thank you for your donation to support Global Change Data Lab – the non-profit organization that helps produce the website Our World in Data.",
+        "Thank you for supporting Our World in Data!",
+        "Your generous donation to Global Change Data Lab, the nonprofit behind Our World in Data, helps us achieve our mission to make data and research on the world’s biggest challenges easier  to understand and use, for everyone: https://ourworldindata.org/problems-and-progress Together, we can make progress against those challenges.",
```

Comment:
```suggestion
        "Your generous donation to Global Change Data Lab, the nonprofit behind Our World in Data, helps us achieve our mission to make data and research on the world’s biggest challenges easier to understand and use, for everyone: https://ourworldindata.org/problems-and-progress Together, we can make progress against those challenges.",
```

Reply:
Should "Together, we can make progress against those challenges." be on a separate line? I feel like having more text after a URL is slightly harder to read.

Reply:
Both changed.